commands/hush.go

```diff
@@ -0,0 +1,87 @@
+package commands
+
+import (
+	"fmt"
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var baseHushDir = filepath.Join(os.TempDir(), "/bendan/hush")
+
+func init() {
+	err := os.MkdirAll(baseHushDir, 0755)
+	if err != nil {
+		log.Println("hush init failed: ", err)
+		return
+	}
+}
+
+func readTimeFromFile(chatID string) (time.Time, error) {
+	filePath := filepath.Join(filepath.Join(baseHushDir, chatID))
```

Comment:
Why do we need `filepath.Join()` twice?

Reply:
It was just a mistake, and I hadn't even found it. XD
I will fix it in the next commit.